### PR TITLE
Adding linters. Setting up the project. Re-creating Awesome books app…

### PR DESCRIPTION
**Pull request by ulises2607**
- Set project.
- Added liters.
- Created modules directory
- Divided code into modules.
- Displayed the current date below the navigation bar with Luxon library

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+git@github.com:ulises2607/Awesome-Books-v2.git
+cd Awesome-Project-v2
+code . (To open VS Code Editor)
+```
+
```
Comment (Efiamotu-1) [inline README.md:91]: - you have omitted the `Usage section` which is to describe how the project can be viewed in the browser by whoever chooses to view it locally. I kindly suggest you make this addititon to your readme 👍 ✨

File: modules/Styles/add-book.css
Comment (Efiamotu-1) [inline modules/Styles/add-book.css:1]: - [x] I kindly suggest you remove all of your styles from the modules directory. Styles doesn't serve the purpose of modules and as such it shouldn't be part of the modules directory 👍

File: modules/JS/books/date.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+// Get the current time in Buenos Aires timezone
+//   const currentTime = DateTime.now().setZone('America/Argentina/Buenos_Aires');
+
+// Format the time
+//   const formatedTime = currentTime.toFormat("MMMM d yyyy',' hh:mm:ss a");
+
+// Display the time in the 'set-time' element
+//   const elementTime = document.getElementById('current-date');
+//   elementTime.innerText = formatedTime;
+// };
```
Comment (Efiamotu-1) [inline modules/JS/books/date.js]: - I kindly suggest you remove every commented code from your code base. It is unprofessional to leave commented code in your code base. 👍

File: modules/JS/books/date.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+// Import necessary elements
+// import { DateTime } from "../../../node_modules/luxon/src/luxon.js";
```
Comment (Efiamotu-1) [inline modules/JS/books/date.js]: - I kindly suggest you remove every commented code from your code base. It is unprofessional to leave commented code in your code base. 👍

File: modules/JS/books/managing-books.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+export const addBook = () => {
+  form.addEventListener('submit', (event) => {
+    event.preventDefault();
+    const title = document.querySelector('#title');
+    const author = document.querySelector('#author');
+
+    storage.createBook(title.value, author.value);
+    storage.displayData();
+  });
+};
```
Comment (Efiamotu-1) [inline modules/JS/books/managing-books.js]: - [ ] I kindly suggest you create separate modules for this function and export it to be used. It is important to break your code into smaller units where each module serves a single purpose 👍

File: modules/JS/books/managing-books.js [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+export const removeBook = () => {
+  bookList.addEventListener('click', (event) => {
+    if (event.target.classList.contains('delete')) {
+      const id = parseInt(event.target.dataset.id, 10);
+      storage.removeData(id);
+    }
+  });
+};
```
Comment (Efiamotu-1) [inline modules/JS/books/managing-books.js]: - [ ] I kindly suggest you create separate modules for this function and export it to be used. It is important to break your code into smaller units where each module serves a single purpose 👍

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @ulises2607 , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart:
- ✔️ Great job implementing date using `luxon` 
- ✔️ Good UI design
- Good job implementing & modularizing your code base
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/JS/books/managing-books.js (hunk 4 above)
Comment (ulises2607) [reply modules/JS/books/managing-books.js]: @Efiamotu-1 Can you give some suggestions for this? because I have a lot of errors trying it

**Review by christianbiring1 — APPROVED** [review]
Hi @ulises2607 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 

## Highlights
- [ ] Great job implementing the changes as suggested in the previous review :+1: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**